<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultadoBusca</class>
 <widget class="QDialog" name="ResultadoBusca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chave</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Conteudo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_chave">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="e_conteudo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_fechar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultado:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_resultado">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
